src/pmdas/amdgpu/drm.c

```
@@ -269,3 +270,157 @@ const char *DRMErrStr(drmReturn_t sts)
+	default:
+	    fprintf(stderr, "Botch: no support for sensor_type=%u\n", sensor_type);
+	    return -EINVAL;
+    }
+    return 0;
+}
+
+int
+amdgpu_query_info(amdgpu_device_handle dev, unsigned info_id, unsigned size, void *value)
+{
+    struct drm_amdgpu_memory_info mem;
+    fprintf(stderr, "fake amdgpu_query_info(%p,%u,%u,%p) called\n", dev, info_id, size, value);
```

Cppcheck (2.21.0)

[warning] Uninitialized variable

(uninitvar)